Contents/Code/siteWowGirls.py

```diff
@@ -0,0 +1,95 @@
+import PAsearchSites
+import PAgenres
+import PAactors
+
+def search(results,encodedTitle,title,searchTitle,siteNum,lang,searchByDateActor,searchDate,searchSiteID):
+    if searchSiteID != 9999:
+        siteNum = searchSiteID
+    searchResults = HTML.ElementFromURL(PAsearchSites.getSearchSearchURL(siteNum) + encodedTitle)
+    index = 0
+    for searchResult in searchResults.xpath('//div[@class="videos-list"]'):
+        titleNoFormatting = searchResult.xpath('.//article/a')[0].text_content().strip()
+        curID = searchResult.xpath('.//article/a')[0].get('href').replace('/','_').replace('?','!')
+        # no Date aviable on Site => taking Date from File
+        releaseDate = searchDate
+        score = 100 - Util.LevenshteinDistance(searchTitle.lower(), titleNoFormatting.lower())
+        results.Append(MetadataSearchResult(id = curID + "|" + str(siteNum), name = titleNoFormatting + " [WowPorn/WowGirls] " + releaseDate, score = score, lang = lang))
+        index += 1
+
+    return results
+
+def update(metadata,siteID,movieGenres,movieActors):
+    Log('******UPDATE CALLED*******')
+
+    url = str(metadata.id).split("|")[0].replace('_','/').replace('?','!')
+    detailsPageElements = HTML.ElementFromURL(url)
+    art = []
+    metadata.collections.clear()
+    movieGenres.clearGenres()
+    movieActors.clearActors()
+
+    # Studio
+    metadata.studio = 'WowGirls'
+
+    # Title
+    metadata.title = detailsPageElements.xpath('//div[@class="title-views"]/h1')[0].text_content().strip().rsplit(" ",3)[0]
+
+    # Summary
+    metadata.summary = detailsPageElements.xpath('//div[@class="desc "]/p')[0].text_content().strip()
+
+    #Tagline and Collection(s)
+    tagline = PAsearchSites.getSearchSiteName(siteID).strip()
+    metadata.tagline = tagline
+    metadata.collections.add(tagline)
+
+    # Genres
+    genres = detailsPageElements.xpath('//div[@class="tags-list"]/a[i[@class="fa fa-folder"]]')
+    if len(genres) > 0:
+        for genreLink in genres:
+            genreName = genreLink.text_content().strip().lower()
+            movieGenres.addGenre(genreName)
+    movieGenres.addGenre("Genre")
```

Comment:
I included this in the Template.py file to demonstrate how to manually add a genre, you'll probably want to remove it.

Reply:
removed